notebooks/analyze_model_output.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<ol><li>Where does this yaml config file come from? </li><li>The validation script needs to be run on training data too or am I missing something?</li><li>Ok so list of images -&gt; config yaml -&gt; model validation -&gt; txt output, is that correct? If yes, do we need to get the list of images from txt output and not just use the original list of images?</li><li>Can this be done using csv?</li><li>Is running the validation script necessary for each country, don't we already have all the predictions for each country just together?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>